envi/__init__.py

```diff
@@ -16,6 +16,7 @@
 ARCH_THUMB16     = 4 << 16
 ARCH_THUMB2      = 5 << 16
 ARCH_MSP430      = 6 << 16
+ARCH_H8          = 7 << 16
```

Comment:
it's well deserved.  it's a horrible architecture (at least the chips i've had to deal with)